btree/btree.c

```diff
@@ -3104,6 +3109,8 @@ static void ff_rec_del_credit(const struct nd *node, m0_bcount_t ksize,
  *
  * Proposed Node Structure for Fix Sized Keys and Variable Sized Value :
  *
+ * Leaf Node Structure :
+ *
  * +-----------+----+-+----+-+----+-+------------------------------------+----+----+----+
  * |           |    | |    | |    | |                                    |    |    |    |
  * |           |    | |    | |    | |                                    |    |    |    |
```

Comment:
Can you add something like V0_off, V_1 off, so that diagram is more clear.

Reply:
okay